### PR TITLE
feature: Add guidance on initial set up of quality gates DOCS-342

### PR DESCRIPTION
**Pull request by prcr**
Configuring Codacy in a way that is perceived as blocking the progress of developers may inadvertently create adoption challenges. This pull request aims to mitigate that by providing guidance on what to consider when configuring quality gates for the first time.

### :eyes: Live preview
https://deploy-preview-1013--docs-codacy.netlify.app/faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate/#configuring-gate

### CODE REVIEW

File: docs/faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ We recommend that you spend some time reviewing the Codacy analysis settings to
 
 [Review and adjust the quality settings](../../repositories-configure/adjusting-quality-settings.md) of your repository to decide which pull requests should fail the Codacy quality gate.
 
+Depending on factors such as the current code quality of your repository and the maturity of your team practices, consider the balance between implementing stricter quality gates and the possibility of blocking or delaying the progress of development. For this reason, Codacy generally recommends that on a first stage you configure rules that focus on stopping new critical issues from entering your code base, such as:
+
+-   High severity issues
+-   Security issues
+-   Considerable drops in code coverage
```
Comment (prcr) [inline docs/faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate.md:50]: @mrfyda @jamescodacy I'd like your opinion on including this guidance here. Do you agree with it? Would you include some more detail (or maybe less)?

Comment (mrfyda) [reply docs/faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate.md:50]: something worth mentioning is that Codacy's default quality settings are designed to help users maintain their current level of code quality (vs improving it). just introducing that might be challenging at first, especially for code coverage.

I think it's also a good reminder for users with uses cases besides standardization.

Comment (prcr) [reply docs/faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate.md:50]: @mrfyda when you mention the default quality settings, are you talking about the default rules that are configured? (In particular, only **New issues are over** is enabled by default.) Or maybe you're talking about the default values for the rules, even the ones that are disabled?

image

Would this be a good way of including that information, or did you have a different rationale in mind? :thinking: 

```suggestion
The default values of the quality gate rules are designed to help maintain the current code quality of your repository. However, depending on factors such as the current code quality of your repository and the maturity of your team practices, consider the balance between implementing stricter quality gates and the possibility of blocking or delaying the progress of development. For this reason, Codacy generally recommends that on a first stage you configure rules that focus on stopping new critical issues from entering your code base, such as:

-   High severity issues
-   Security issues
-   Considerable drops in code coverage
```

Comment (mrfyda) [reply docs/faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate.md:50]: > you're talking about the default values for the rules, even the ones that are disabled
this ☝️ 
specifically, the default value for the coverage quality gate might be demanding

Comment (prcr) [reply docs/faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate.md]: Thanks for clarifying @mrfyda, I understand the implication better now.

I tried to chunk the information a bit better by having the general discussion on the balance between stricter rules and team development speed, and then the particular advice on how to set up the rules in a "Tip" admonition:

image

Comment (prcr) [reply docs/faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate.md]: @mrfyda I'm merging this pull request now, but do let me know if you have any further feedback and we can do some more tweaks. :rocket:

**Pull request merged**